src/woo_search/search_index/api/serializers/publications.py

```diff
@@ -51,12 +51,26 @@ class DocumentSerializer(serializers.Serializer):
         )
     )
     identifier = serializers.CharField(
-        help_text=_("The (primary) unique identifier."),
+        help_text=_("**DEPRECATED** The (primary) unique identifier."),
         max_length=255,
         required=False,
         allow_blank=True,
         default="",
     )
+    identifiers = serializers.ListField(
+        help_text=_("The document identifiers attached to this document."),
+        child=serializers.CharField(
+            max_length=40,
```

Comment:
this will need to be 255 since the original identifier field allowed up to this amount

```suggestion
            max_length=255,
```